geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert2/transforms/MiscFunctionFactoryTest.scala

```diff
@@ -0,0 +1,50 @@
+package org.locationtech.geomesa.convert2.transforms
+
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class MiscFunctionFactoryTest extends Specification {
+
+  "MiscFunctionFactory" >> {
+    "intToBoolean" should {
+      "convert a 0 to false" >> {
+        MiscFunctionFactory.intToBoolean(Array(0)) mustEqual false
```

Comment:
FYI, you should be able to test functions through the Expression interface, as in https://github.com/locationtech/geomesa/blob/master/geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert2/transforms/ExpressionTest.scala#L42-L43. But the tests here are fine, too.